core/interfaces/rebase.py

```diff
@@ -965,10 +965,12 @@ class GsRebaseOnTopOfCommand(GsRebaseDefineBaseRefCommand):
         ["select_ref", "Rebase to ref"],
     ]
 
-    def rebase_to_base_ref(self, selection):
+    def rebase_to_base_ref(self):
         self.set_base_ref(self.view.settings().get("git_savvy.rebase.base_ref"))
 
     def set_base_ref(self, selection):
+        if not selection:
+            return
```

Comment:
I don't think `set_base_ref` is ever called directly from a panel, I couldn't reproduce the issue with selection being missing here ... can you provide reproduction step by step instructions please?